packages/api/package.json

```diff
@@ -34,6 +37,7 @@
   },
   "files": [
     "dist",
+    "legacy-require-handler.**",
```

Comment:
does the `files` array support globs like this?

Reply:
![CleanShot 2023-10-30 at 12 19 59@2x](https://github.com/readmeio/api/assets/8854718/adcf3136-b240-40dd-90aa-88ec133a2824)

yep, confirmed that `npm pack` includes it